java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1447,6 +1448,29 @@ public T zrankWithScore(@NonNull String key, @NonNull String member) {
         return getThis();
     }
 
+    /**
+     * Removes all elements in the sorted set stored at <code>key</code> with rank between <code>start
+     * </code> and <code>end</code>. Both <code>start</code> and <code>end</code> are zero-based
+     * indexes with <code>0</code> being the element with the lowest score. These indexes can be
+     * negative numbers, where they indicate offsets starting at the element with the highest score.
```

Comment:
Is this terminology correct? I feel like score and rank aren't really the same thing. I guess technically the first element has the lowest score and the last has the highest score, but there could be multiple values with the same lowest/highest score.

Reply:
That's right, `rank` != `score`. `rank` is like an element position (integer) and `score` is a weight (double).

Reply:
Let's push this upstream first and then we can discuss with AWS team about changing this.